Requirements/ViewModels/RequirementBrowser/Rows/SimpleParameterValueRowViewModel.cs

```diff
@@ -82,7 +82,19 @@ public string Name
         /// </summary>
         public string ShortName
         {
-            get { return this.shortName; }
+            get
+            {
+
+                if (this.Scale != null)
+                {
+                    return string.Concat(this.shortName, "(", this.Scale.ShortName, ")");
```

Comment:
Should be " [" (note the space) ...... "]".

Reply:
It does react on both short name and scale changes

Reply:
ok, good, can you still change this line to 

```
return $"{this.shortName} [{this.Scale.ShortName}]";
```

Reply:
agree with alex